orc8r/gateway/python/magma/common/redis/containers.py

```diff
@@ -422,5 +424,16 @@ def lock(self, key: str) -> Lock:
             strict=False,
         )
 
+    def _sync_cache(self):
+        """
+        Syncs write-through cache with redis data on store.
+        """
+        type_pattern = "*:" + self.redis_type
```

Comment:
we should probably have a function called `get_type_pattern() -> str:` so that we make sure we're consistent everywhere.